README.md

```diff
@@ -306,7 +306,7 @@ var mergedResult = results.Merge();
 
 ### Converting
 
-A result object can be converted to another result object with methods `ToResult()` and `ToResult<TValue>()`.
+A result object can be converted to another result object with methods `ToResult()`, `ToResult<TValue>()`, `Bind()` and `Bind<TValue>()`.
```

Comment:
I think Bind() is not a method about converting. Please create a new headline for the Bind() stuff.

Reply:
Added in a new section